jupyter-js-widgets/.npmignore

```diff
@@ -5,15 +5,10 @@ _build
 docs/man/*.gz
 docs/source/api/generated
 docs/gh-pages
-static/components/
-static/notebook/js/
```

Comment:
In my opinion, the content of `static/notebook/js` should still be excluded from the `jupyter-js-widgets` npm package and not moved with the rest of the Javascript.